scripts/deploy.ts

```diff
@@ -92,9 +92,9 @@ async function main() {
   console.log('Contract address:', address)
   console.log(
     'Etherscan URL:',
-    `https://${
-      chainName !== 'mainnet' ? `${chainName}.` : ''
-    }etherscan.io/address/${address}`
+    `https://${chainName !== 'mainnet' ? `${chainName}.` : ''}${
```

Comment:
Let's extract this into separate function, also, change the check to be:
```TS
${chainName === 'mainnet' ? '' : chainName}
```
We don't need double nested `${${}}` here